app/my_flask.py

```diff
@@ -28,3 +32,49 @@ def __init__(self, import_name: str):
 
     def preprocess_request(self):
         super().preprocess_request()
+        if '/ping' in request.path:
+            return
+
+        site_id = getattr(config, 'MATOMO_SITE_ID', 1)
+        if not site_id:
```

Comment:
What is the purpose of this check?

Reply:
Default value removed, it should not be sent if the SITE ID and SIT URL are not available.